src/containers/ProjectContentsContainer.jsx

```diff
@@ -103,6 +106,9 @@ class ProjectContentsContainer extends Component {
       actions.updateTranslation(translation, field, translationText);
     }
     this.closeModal();
+    this.setState({
+      translationText: '',
```

Comment:
This means that the new translation will never display in the editor, right? That's wrong.